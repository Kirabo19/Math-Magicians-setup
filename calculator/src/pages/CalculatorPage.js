import React from 'react';
import Calculator from '../components/Calculator';

const CalculatorPage = () => (
  <div className="flex calculator_content">
    <h1>Let&apos;s do some math!</h1>
    <Calculator />
  </div>
);

export default CalculatorPage;
