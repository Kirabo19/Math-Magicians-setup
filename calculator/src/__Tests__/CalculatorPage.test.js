import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorPage from '../pages/CalculatorPage';

it('test calculator', () => {
  const calculator = renderer
    .create(<CalculatorPage />)
    .toJSON();
  expect(calculator).toMatchSnapshot();
});