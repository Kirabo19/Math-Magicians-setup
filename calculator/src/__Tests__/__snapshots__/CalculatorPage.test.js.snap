// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test calculator 1`] = `
<div
  className="flex calculator_content"
>
  <h1>
    Let's do some math!
  </h1>
  <div
    className="calculator"
  >
    <div
      className="output"
      data-testid="output"
    >
      0
    </div>
    <div
      className="numbers"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="number0"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        .
      </button>
    </div>
    <button
      className="operation"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      className="operation"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      className="operation"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      className="operation"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      className="operation"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
    <button
      className="options"
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      className="options"
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      className="options"
      onClick={[Function]}
      type="button"
    >
      %
    </button>
  </div>
</div>
`;
